experimenter/experimenter/nimbus-ui/src/components/AppLayoutSidebarLaunched/index.tsx

```diff
@@ -235,7 +235,7 @@ export const AppLayoutSidebarLaunched = ({
                     "Experiment analysis was skipped"
                   ) : (
                     <>
-                      Experiment analysis not ready yet.
+                      Analysis results are not available for rollouts.
```

Comment:
Oh you'll want to actually check if it's a rollout here.  It's an experiment then `Experiment analysis not ready yet` is correct, but if we land here and it's a rollout then the new message is correct.